src/index.js

```diff
@@ -1,7 +1,295 @@
+                  <option>Rutland</option>
+                  <option>Washington</option>
+                  <option>Windham</option>
+                  <option>Windsor</option>
+                </select>
+              </label></p>
+              <menu>
+                <button value="cancel">Cancel</button>
+                <button id="confirmBtn" value="default" onClick={this.props.handleCountySelect}>OK</button>
+              </menu>
+            </form>
+          </dialog>
+        </div>
+      )
+    }
+  }
+}
+
+class Info extends React.Component {
+  render() {
+    if (this.props.upGiven) {
+      return (
+        <div id="info">
+          <p>Latitude: {this.props.randomLocation.lat}</p>
+          <p>Longitude: {this.props.randomLocation.lng}</p>
+          <p>County: {this.props.randomLocation.county}</p>
+          <p>Municipality: {this.props.randomLocation["municipalityName"]}</p>
+          <p>Score: {this.props.score}</p>
+        </div>
+      )
+    } else if (this.props.countyGuess) {
+        console.log("guess: " + this.props.countyGuess)
+        let guessWithCounty = this.props.countyGuess + " County"
+        if (guessWithCounty===this.props.randomLocation.county) {
+          return (
+            <div id="info">
+              <p>Latitude: {this.props.randomLocation.lat}</p>
+              <p>Longitude: {this.props.randomLocation.lng}</p>
+              <p>County: {this.props.randomLocation.county}</p>
+              <p>Municipality: {this.props.randomLocation["municipalityName"]}</p>
+              <p>You were correct!</p>
+              <p>Score: {this.props.score}</p>
+            </div>
+          )
+        } else {
+          return (
+            <div id="info">
+              <p>Latitude: {this.props.randomLocation.lat}</p>
+              <p>Longitude: {this.props.randomLocation.lng}</p>
+              <p>County: {this.props.randomLocation.county}</p>
+              <p>Municipality: {this.props.randomLocation["municipalityName"]}</p>
+              <p>You were incorrect :(</p>
+              <p>Score: {this.props.score}</p>
+            </div>
+          )
+        }
+    } else {
+      return (
+        <div id="info">
+          <p>Latitude: ?</p>
+          <p>Longitude: ?</p>
+          <p>County: ?</p>
+          <p>Municipality: ?</p>
+          <p>Score: {this.props.score}</p>
+        </div>
+      )
+    }
+  }
+}
+
+class Controls extends React.Component {
+  render () {
+    return (
+      <div id="controls">
+        <h2 style={{gridArea: "title"}}>Controls:</h2>
+        <button id="up" style={{gridArea: "up"}}>↑</button>
+        <button id="left" style={{gridArea: "left"}}>←</button>
+        <button id="right" style={{gridArea: "right"}}>→</button>
+        <button id="down" style={{gridArea: "down"}}>↓</button>
+      </div>
+    )
+  }
+}
 
 ReactDOM.render(
-  <h1>Hello World!</h1>,
+  <Page />,
   document.getElementById('root')
-)
\ No newline at end of file
+)
+
```

Comment:
consider moving these functions inside your class components. Not only will it make your code more organized, but it will also prevent other components from calling them when they shouldn't.